microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -85,10 +85,27 @@ After you launch Microsoft Edge \(Chromium\) using a command-line argument, IE m
 
 IE mode remains unavailable until you completely close and restart Microsoft Edge \(Chromium\).  
 
+
+## Replacing IE Automation
+
+If you have an existing application that uses the [InternetExplorer][InternetExplorerObject] object to automate Internet Explorer 11, your application will not work if the Internet Explorer 11 desktop application is not available.  Internet Explorer 11 will be retired on June 15, 2022. For more information, navigate to [The future of Internet Explorer on Windows 10 is in Microsoft Edge](https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/).  Microsoft Edge \(Chromium\) does not support automating IE mode through the [InternetExplorer][InternetExplorerObject] object, so applications that rely on this object will need to be updated using one of the recommended alternatives below.  Applications that use the [WebBrowser][WebBrowserControl] control will continue to work and will not be affected by the removal of Internet Explorer 11.
+
+If your automation application does not require IE mode for the website \(or app\) content to function correctly, we recommend updating your application to use Microsoft Edge \(Chromium\) instead of Internet Explorer 11.  Many available automation tools support Microsoft Edge \(Chromium\), including WebDriver and Playwright.  To learn more about automating Microsoft Edge \(Chromium\) using WebDriver, navigate to [WebDriver overview][WebDriverIndex].  To learn more about using Playwright, navigate to [Playwright overview][PlaywrightIndex].
```

Comment:
```suggestion
If your automation application doesn't require IE mode for the website \(or app\) content to function correctly, we recommend updating your application to use Microsoft Edge \(Chromium\) instead of Internet Explorer 11.  Many available automation tools support Microsoft Edge \(Chromium\), including WebDriver and Playwright.  To learn more about automating Microsoft Edge \(Chromium\) using WebDriver, navigate to [WebDriver overview][WebDriverIndex].  To learn more about using Playwright, navigate to [Playwright overview][PlaywrightIndex].
```